Scenes/Spaceship/ExplosionParticleSystem.gd

```diff
@@ -0,0 +1,11 @@
+extends Node2D
+
+func _ready():
+	self.set_physics_process(true)
```

Comment:
This is not needed for the particle system to work.

`set_physics_process(true)` activates the `_physics_process(delta)` method call. https://docs.godotengine.org/en/3.1/classes/class_node.html#class-node-method-physics-process

Reply:
Yeah. I think I forgot to delete that code from some experiments I did :D